### ISSUE TITLE
[BUG] bro i applied your patch but this new errors popped in

### ISSUE THREAD

**Issue by silverbullet-herr** [dedshit/metasploit_in_termux#10]
![Screenshot_20220924-163027_Termux](https://user-images.githubusercontent.com/64875326/192106355-c3d792e3-de01-4eae-82ab-e8da5e8a2a80.png)
![Screenshot_20220924-163023_Termux](https://user-images.githubusercontent.com/64875326/192106374-9bfa4da8-7f99-4b16-9774-da33dbccc131.png)
![Screenshot_20220924-163017_Termux](https://user-images.githubusercontent.com/64875326/192106383-926d1f70-5cec-4700-90b3-f7420ab4d411.png)

**Comment by dedshit** [comment]
binutils installed ?

**Comment by silverbullet-herr** [comment]
> 

yeah it is , i think the framework  is required to work  with that specific version of nokogiri "1.8.0" not that latest one , but i can't install that version

**Comment by dedshit** [comment]
> > 
> 
> yeah it is , i think the framework is required to work with that specific version of nokogiri "1.8.0" not that latest one , but i can't install that version

no its not

**Comment by dedshit** [comment]
try this `bundle config build.nokogiri "--use-system-libraries --with-xml2-include=$PREFIX/include/libxml2"; bundle install`

**Comment by silverbullet-herr** [comment]
all the bundles are installed but gives the same error

**Comment by dedshit** [comment]
> all the bundles are installed but gives the same error

your bundler version

**Comment by dedshit** [comment]
> all the bundles are installed but gives the same error

send ss of detailed log that you have installed

**Comment by silverbullet-herr** [comment]
after this command ""bundle config build.nokogiri "--use-system-libraries --with-xml2-include=$PREFIX/include/libxml2"; bundle install ""
i get :


![Screenshot_20220924-191624_Termux](https://user-images.githubusercontent.com/64875326/192113051-8da593f8-4acb-45eb-9929-cd598bd880b7.png)
![Screenshot_20220924-191631_Termux](https://user-images.githubusercontent.com/64875326/192113054-aa8c2930-f64f-4b23-bdc2-251f7acea9fb.png)

and when i type msfconsole i get  the same error

**Comment by dedshit** [comment]
> after this command ""bundle config build.nokogiri "--use-system-libraries --with-xml2-include=$PREFIX/include/libxml2"; bundle install "" i get :
> 
> ![Screenshot_20220924-191624_Termux](https://user-images.githubusercontent.com/64875326/192113051-8da593f8-4acb-45eb-9929-cd598bd880b7.png) ![Screenshot_20220924-191631_Termux](https://user-images.githubusercontent.com/64875326/192113054-aa8c2930-f64f-4b23-bdc2-251f7acea9fb.png)
> 
> and when i type msfconsole i get the same error

try install bundler latest version &try again

**Comment by silverbullet-herr** [comment]
> try install bundler latest version &try again
it is the latest bundler --version = 
2.3.22
it is just one file that ruined the whole metasploit 
nokogiri 1.8.0 is unsolveable

**Comment by dedshit** [comment]
try to run nokogiri 

> > try install bundler latest version &try again
> > it is the latest bundler --version =
> > 2.3.22
> > it is just one file that ruined the whole metasploit
> > nokogiri 1.8.0 is unsolveable

still same problem ?

**Comment by silverbullet-herr** [comment]
> try to run nokogiri
> 
> > > try install bundler latest version &try again
> > > it is the latest bundler --version =
> > > 2.3.22
> > > it is just one file that ruined the whole metasploit
> > > nokogiri 1.8.0 is unsolveable
> 
> still same problem ?
> 
> ![Screenshot_20220925-004455_Termux](https://user-images.githubusercontent.com/62318734/192114837-208c3fcd-2b6e-4be4-b7ac-c974086b1ecd.png)

can you run msf with that version ? 
mine didn't work 
![Uploading Screenshot_20220924-201952_Termux.png…]()

**Comment by dedshit** [comment]
screenshot not properly uploaded

**Comment by silverbullet-herr** [comment]
> try to run nokogiri
> 
> > > try install bundler latest version &try again
> > > it is the latest bundler --version =
> > > 2.3.22
> > > it is just one file that ruined the whole metasploit
> > > nokogiri 1.8.0 is unsolveable
> 
> still same problem ?


i was managed to install nokogiri latest but didn't work with gushmazuku installation cuz it had to be 1.8.0 in that installation maybe it could work with yours?

**Comment by silverbullet-herr** [comment]
> screenshot not properly uploaded

![Uploading Screenshot_20220924-201952_Termux.png…]()

**Comment by dedshit** [comment]
still same

**Comment by dedshit** [comment]
wait until file properly uploads

**Comment by silverbullet-herr** [comment]
> screenshot not properly uploaded

![Screenshot_20220924-201952_Termux](https://user-images.githubusercontent.com/64875326/192115614-d18cd3fa-a38c-472e-b0e2-6f5f190fb151.png)

**Comment by dedshit** [comment]
if nokogiri properly installed then there will be a directory named nokogiri-1.13.8

**/data/data/com.termux/files/usr/lib/ruby/gems/3.1.0/gems/nokogiri-1.13.8**

**Comment by silverbullet-herr** [comment]
> 

i have this 
![Screenshot_20220924-210420_Termux](https://user-images.githubusercontent.com/64875326/192116508-b01f5140-f512-474e-ab4f-d2dfa2e100cc.png)

**Comment by dedshit** [comment]
> if nokogiri properly installed then there will be a directory named nokogiri-1.13.8
> 
> **/data/data/com.termux/files/usr/lib/ruby/gems/3.1.0/gems/nokogiri-1.13.8**

check for this directory

**Comment by dedshit** [comment]
> > if nokogiri properly installed then there will be a directory named nokogiri-1.13.8
> > **/data/data/com.termux/files/usr/lib/ruby/gems/3.1.0/gems/nokogiri-1.13.8**
> 
> check for this directory also try running `nokogiri` command

**Comment by dedshit** [comment]
@silverbullet-herr  
![20220925_124903](https://user-images.githubusercontent.com/62318734/192132650-14ea76a4-1087-4b85-ba75-ae459923ca30.png)

try to remove that aarch64 version gem & old one 1.8.0

**Comment by Apxcore** [comment]
![Uploading Screenshot_2022-09-25-10-58-41-391_com.termux.jpg…]()
Pls help

**Comment by silverbullet-herr** [comment]
> @silverbullet-herr ![20220925_124903](https://user-images.githubusercontent.com/62318734/192132650-14ea76a4-1087-4b85-ba75-ae459923ca30.png)
> 
> try to remove that aarch64 version gem & old one 1.8.0

@dedshit i finally managed to lauch it succefully !!
im not really sure what i was doing cuz i don't know much about ruby but here the steps i made:
i removed nokogiri using : gem remove nokogiri 
i reinstalled it using : gem install nokogiri --platform=ruby
using the same method to install and reinstall sqlite3 cuz it gave me the same error
now it's working perfectly :)

**Comment by silverbullet-herr** [comment]
@Apxcore can't see your screen shot

**Comment by dedshit** [comment]
> > @silverbullet-herr ![20220925_124903](https://user-images.githubusercontent.com/62318734/192132650-14ea76a4-1087-4b85-ba75-ae459923ca30.png)
> > try to remove that aarch64 version gem & old one 1.8.0
> 
> @dedshit i finally managed to lauch it succefully !! im not really sure what i was doing cuz i don't know much about ruby but here the steps i made: i removed nokogiri using : gem remove nokogiri i reinstalled it using : gem install nokogiri --platform=ruby using the same method to install and reinstall sqlite3 cuz it gave me the same error now it's working perfectly :)

thanks for the feedback

**Comment by Apxcore** [comment]
> @Apxcore can't see your screen shot

![Uploading Screenshot_2022-09-25-10-58-41-391_com.termux.jpg…]()

**Comment by silverbullet-herr** [comment]
> 

it's not uploaded properly 
wait for it to upload then poste it

**Comment by Apxcore** [comment]
Your patch worked 

> still same

Your patch worked for me after few try and force stop termux thank you and for that I will give you a star
